modules/standard/Time.chpl

```diff
@@ -1326,7 +1326,10 @@ module Time {
                       second=second, microsecond=microsecond);
   }
 
-  /* Get the `time` portion of the `dateTime` value, with `tz` = nil */
+  /* Get the `time` portion of the `dateTime` value, with ``tz=nil``.
+     In the future `timezone` interactions with extracting the :type:`time`
```

Comment:
Does this warrant an `@unstable` since the docs say we may change this?

Reply:
I was actually wondering about that. In our closing comment on the discussion issue I had written down "put a note (not unstable) in time() docs about future potential changes with timezone handling". But I can't actually remember the motivation for that. At the moment it's not making sense to me as I don't think we'd be changing this method when we change timezone handling.

Reply:
I believe the context was we were discussing was about providing a second overload that is tz aware? At the very least, I think the return type may change when tz is apart of the type (but this applies to a lot of procedures)

Reply:
There is already a TZ aware version `timetz()` (unstable) -- I can't imagine we'd do more than maybe rename either of these methods, or combine them and make tz awareness a parameter which would be an implementation detail. That wouldn't warrant `unstable` or even this docs note IMO.

Reply:
Per offline conversation, going to remove this note